submissions/israelOladosu/introductionToHtml/index.html

```diff
@@ -2,65 +2,79 @@
 <html lang="en">
```

Comment:
You made this same changes in your pull request for [8]. This will cause conflict when both pull request are merged and you are updating your files. You need to separate the changes you are making  in different pull request.